components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -380,11 +388,35 @@ protected boolean handlePostAuthentication(Event event, String userName, UserSto
                          */
                         unlockTimePropertyValue = (long) (unlockTimePropertyValue * 1000 * 60 * Math.pow
                                 (unlockTimeRatio, currentFailedLoginLockouts));
+                        accountLockDuration = unlockTimePropertyValue/60000;
                         unlockTime = System.currentTimeMillis() + unlockTimePropertyValue;
                         newClaims.put(AccountConstants.ACCOUNT_UNLOCK_TIME_CLAIM, Long.toString(unlockTime));
                     }
                 }
                 currentFailedLoginLockouts += 1;
+
+                if (currentFailedLoginLockouts > 1) {
+                    boolean notificationOnLockIncrement = false;
+                    try {
+                        notificationOnLockIncrement = Boolean.parseBoolean(AccountUtil.getConnectorConfig(AccountConstants
+                                .NOTIFY_ON_LOCK_DURATION_INCREMENT, tenantDomain));
+                    } catch (IdentityEventException e) {
+                        log.warn("Error while reading notification on lock increment property in account lock handler. "
+                                + e.getMessage());
+                        if (log.isDebugEnabled()) {
+                            log.debug("Error while reading notification on lock increment property in account lock " +
+                                    "handler", e);
+                        }
+                    }
```

Comment:
extract to a private method

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/95/commits/d02b5bf3cdadeca3967fa0d1aca174e5e2240de5